source/classes.tex

```diff
@@ -739,15 +739,11 @@
 \pnum
 \begin{note}
 \indextext{layout!class object}%
-Non-static data members of a (non-union) class
-with the same access control\iref{class.access} and
+Non-variant non-static data members of
 non-zero size\iref{intro.object}
 are allocated so that later
 members have higher addresses within a class object\iref{expr.rel}.
 \indextext{allocation!unspecified}%
```

Comment:
@jensmaurer: should this index entry go away, too? I can do that during rebasing, just let me know.

Reply:
Yes, that should probably go away.

Reply:
Thanks!